extensions-core/protobuf-extensions/pom.xml

```diff
@@ -45,6 +46,20 @@
     </repository>
   </repositories>
 
+  <dependencyManagement>
+    <dependencies>
+      <!-- This is an indirect dependency of kafka-protobuf-provider
+      update to address vulnerability in transitive dependency okio  -->
+      <dependency>
+          <groupId>com.squareup.okio</groupId>
+          <artifactId>okio</artifactId>
+          <version>${okio.version}</version>
+          <scope>provided</scope>
```

Comment:
we shouldn't be overriding the scope here, we should let the dependencies decide on the scope.
```suggestion
```

Reply:
That dependency is in that scope already, this just overrides the version. See: 
https://github.com/apache/druid/pull/15449#discussion_r1421790202